### DISCUSSION TITLE
Using Container SDK & Visual Studio `.dcproj` Compose Support

### DISCUSSION THREAD

**Discussion by MeikTranel** [dotnet/sdk-container-builds#591] [Q&A]
My team has been using the old multistage build dockerfile in combination with the visual studio compose file support to set up a dev environment. Is it possible to wire the dcproj back up with the container sdk and if not is it planned?

**Comment by baronfel** [comment]
There currently isn't any native integration with `dcproj` - for that you'd need to file a feature request for `dcproj` team over at [Visual Studio Developer Community](https://developercommunity.visualstudio.com/VisualStudio?q=dcproj). There are potentially a few things on our side that could make that integration easier, but they aren't high priorities for us compared to core functionality like multi-manifest publishing at this time.
